### DISCUSSION TITLE
ISO Question

### DISCUSSION THREAD

**Discussion by TWDesigns** [automatic-ripping-machine/automatic-ripping-machine#974] [Q&A] [answered]
When I first started converting my movie collection to digital so that I could stream from one location I for some reason choose ISO. Looking back the only thing I could thing of is in case I wanted to remake the original. So my question is, will this application batch process folders that contains ISOs to a different format? I read about going to ISO but I want to come from ISO.

Thanks in advance for any input!

**Comment by shitwolfymakes** [comment, marked as answer]
Currently we don't have a way to do that, but it's on todo list
